packages/eslint-plugin/lib/rules/unified-signatures.js

```diff
@@ -0,0 +1,616 @@
+/**
+ * @fileoverview Warns for any two overloads that could be unified into one by using a union or an optional/rest parameter.
+ * @author Armando Aguirre
```

Comment:
Sorry we've changed to using all-contributors now and removed all other occurrences of these file-level author comments, please could you remove this

Reply:
Removed.